manifests/e/EaglePointSoftware/PinnacleSeriesManagementUtility/2023.2.1.101/EaglePointSoftware.PinnacleSeriesManagementUtility.locale.en-US.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: EaglePointSoftware.PinnacleSeriesManagementUtility
+PackageVersion: 2023.2.1.101
+PackageLocale: en-US
+Publisher: Eagle Point Software
+PublisherUrl: https://www.eaglepoint.com/
+PublisherSupportUrl: https://support.eaglepoint.com/
+PrivacyUrl: https://www.eaglepoint.com/privacy-policy/
+Author: Eagle Point Software Corporation
+PackageName: Pinnacle Series Management Utility
```

```suggestion
PackageName: Pinnacle Series
```
This should match control panel